src/gui/windows/manage_players.zig

```diff
@@ -22,12 +22,30 @@ pub var window = GuiWindow{
 const padding: f32 = 8;
 var userList: []*main.server.User = &.{};
 
+pub var needsUpdate: bool = false;
+
 fn kick(conn: *main.network.Connection) void {
 	conn.disconnect();
 }
 
+fn ipBan(conn: *main.network.Connection) void {
+	const ip = conn.remoteAddress.ip;
+	main.server.Settings.ipBanList.append(ip);
+	conn.disconnect();
+	needsUpdate = true;
+}
+
+// u32 in a pointer costume
```

Comment:
Instead of `u32` I think it would better if you made a version of `network.Address` without the port.